internal/tiltfile/docker.go

```diff
@@ -286,6 +285,21 @@ func (s *tiltfileState) customBuild(thread *starlark.Thread, fn *starlark.Builti
 		return nil, errors.Wrap(err, "live_update")
 	}
 
+	// parsing ignore values to strings to be added to dockerimage struct
+	tempIgnores := starlarkValueOrSequenceToSlice(ignoreVal)
```

Comment:
I noticed that this code is exactly the same as the code on [L173](https://github.com/windmilleng/tilt/blob/6ce179c49633b3aa4a8185df54e3a3ee980ad961/internal/tiltfile/docker.go#L173). Should we factor it out in to a function?